potpie/context-engine/adapters/inbound/cli/auth/auth_commands.py

```
@@ -215,7 +238,38 @@ def _run_linear_oauth_flow(*, force: bool = False) -> None:
+                    {
+                        "ok": True,
+                        "already_connected": True,
+                        "provider": "linear",
+                        "workspaces": orgs,
+                        "hint": "Run with --add to connect another Linear workspace.",
+                    },
+                    as_json=True,
+                )
+                return
+            print_plain_line(
+                f"Linear is already connected"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)